Library/PaddingLabel.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+public typealias PaddingLabelStyle = (PaddingLabel) -> PaddingLabel
+
+public class PaddingLabel: UILabel {
```

Comment:
Could you tell me a bit about why we are using this custom UILabel instead of applying a style to a UILabel?

Reply:
Oh yeah. Intrinsic content size of a label doesn't have padding, if you add a background colour it won't look like the design,  so to make the AuthorBadge look like the design with the padding I'll need to subclass UILabel.